.github/workflows/build-images.yml

```diff
@@ -65,8 +75,14 @@ jobs:
         echo "Status:    ${{ steps.buildx.outputs.status }}"
         echo "Flags:     ${{ steps.buildx.outputs.flags }}"
         echo "Platforms: ${{ steps.buildx.outputs.platforms }}"
-    - name: Build ${{ matrix.name }}
-      run: mvn install --projects ${{ matrix.modules }} --also-make -DskipTests ${{ matrix.args }}
+    - name: Build ${{ matrix.name }} on ${{ matrix.ubi }}
+      run:
+        mvn install
+          --projects ${{ matrix.modules }}
+          --also-make
+          -DskipTests
+          ${{ matrix.args }}
+          ${{ steps.prepare.outputs.ubi-args }}
```

Comment:
Does it read $GITHUB_OUTPUT? (sorry, not an action expert)

Reply:
Yes. https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-output-parameter